metacat-common-server/src/main/java/com/netflix/metacat/common/server/monitoring/Metrics.java

```diff
@@ -28,89 +28,100 @@
     AppPrefix("metacat"),
 
     /**
+     * Measure types
+     */
+    Count("count"),
+    Gauge("gauge"),
+    Timer("timer"),
+
+    /**
      * evnets.
      */
-    CounterEventAsync(AppPrefix + ".events.count.Async"),
-    CounterEventSync(AppPrefix + ".events.count.Sync"),
+    Events("events"),
+    CounterEventAsync(Name(Events, Count , "async")),
+    CounterEventSync(Name(Events, Count, "sync")),
 
     /**
      * thrift request.
      */
-    CounterThrift(AppPrefix + ".thrift.count.request"),
+    Thrift("thrift"),
+    CounterThrift(Name(Thrift, Count,   "request")),
 
     /**
      * hive sql lock error.
      */
-    CounterHiveSqlLockError(AppPrefix + ".count.hiveSqlLockError"),
-    CounterHiveExperimentGetTablePartitionsFailure(AppPrefix + ".hive.count.experimentGetPartitionsFailure"),
+    Hive("hive"),
```

Comment:
I disagree. Hive connector is a plugin/spi implementation and shouldn't be leaking into the main code base. If we're keeping it here just for enum convenience then that seems to be a bad design and this should be class that can be extended. @ajoymajumdar have spoken before about the need to remove this kind of leakage.